libcxx/docs/ReleaseNotes/21.rst

```diff
@@ -55,6 +55,8 @@ Improvements and New Features
 Deprecations and Removals
 -------------------------
 
+- ``std::is_pod`` and ``std::is_pod_v`` are deprecated since C++20 according to the standard.
```

Comment:
Can you add `P0767R1` to the Implemented paper section? And as you suggested update the C++20 paper status page too.

I feel the "according to the standard"  superfluous since that's true for most things we do. WDYT?
```suggestion
- ``std::is_pod`` and ``std::is_pod_v`` are deprecated since C++20.
```

Reply:
I think I'd rather say something like "is_pod and is_pod_v are now marked as deprecated in C++20 and later". That makes it clear that we added annotations.